dangerfile.ts

```diff
@@ -2,22 +2,23 @@ import { execSync } from 'child_process';
 import { danger, message, warn } from 'danger';
 import { readFileSync, statSync } from 'fs';
 
+// Gather changes
+const modifiedFiles = danger.git.modified_files ?? [];
+const modifiedSourceFiles = modifiedFiles.filter((path) => /\/src\/.+\.tsx?/.exec(path));
+
```

Comment:
```suggestion

if (!modifiedFiles.length) {
  warn('No files were modified by this PR. Make sure this is expected!'); 
}
```
I think in most cases this is indication that something is not right. This should almost never happen and I think the failure might actually be warranted. Could be that it's not actually running on the right branch.

Reply:
Maybe this should be a fail and not a warn too

Reply:
Yes, agree, I think this implies something isn't configured correctly.

My short term goal is to just stabilize this and get it working consistently again.

Longer term, my honeymoon period with Danger.js is over, and I'm not sure this is the right tool for us.  I like the idea of having something like this for medplum specific checks where we can catch common issues.  There might be a simpler option for that though.